client/web/src/auth/useExternalServices.ts

```diff
@@ -28,9 +31,12 @@ interface UseExternalServicesResult {
     ) => Promise<ApolloQueryResult<ExternalServicesResult>>
 }
 
-export const useExternalServices = (userId: string | null): UseExternalServicesResult => {
+export const useExternalServices = (
+    userId: string | null,
+    fetchCollaborators: boolean = false
+): UseExternalServicesResult => {
     const { data, loading, error, refetch } = useQuery<ExternalServicesResult, ExternalServicesVariables>(
```

Comment:
I think this should be something like:

```suggestion
    const { data, loading, error, refetch } = useQuery<ExternalServicesResult | ExternalServicesWithCollaboratorsResult, ExternalServicesVariables | ExternalServicesWithCollaboratorsVariables>(
```

But this might also be screwing with the typing as it now can't know for sure which variables set to use for which result set (both variables are the same ATM but they may converge) so perhaps we should split this into two different useQueries?